src/components/Container.js

```diff
@@ -37,9 +36,8 @@ export const Container = () => {
           </main>
         </div>
         <div id="map">
-          <Map_Filter />
-          <Map />
-          {/* <CollectionsPage /> */}
+          <MapFilter />
+          {/* <Map /> */}
```

Comment:
Please remember: No commented out code should be included in pushes to the main branch. If you want to see your progress in production, use /features syntax in your branch naming and AWS will create a deployment of your branch for you.

Reply:
Got it. Fixing now.